src/render.tsx

```diff
@@ -22,19 +24,26 @@ export type RenderResult = ReturnType<typeof render>;
  * Renders test component deeply using react-test-renderer and exposes helpers
  * to assert on the output.
  */
-export default function render<T>(
+export function render<T>(
```

Comment:
This should be wrapped behind an `render` option, which should be disabled by default at least until next major release as that be a breaking change. Let's figure out name of that option:
`validateRenderedStrings`, `validateStringsRenderedInText`, `validateStringsOutsideText`, etc, etc?

Reply:
I think validateRenderedStrings is good, should I also rename the method assertStringsWithinText ? Maybe validateStringsRenderedInText since it's an internal name it could be a bit longer but I find it more explicit

Reply:
`validateRenderedStrings` seems ambiguous, there is no mention about them being outside of `Text`